packages/model-viewer/src/three-components/gltf-instance/ModelViewerGLTFInstance.ts

```diff
@@ -107,8 +107,7 @@ export class ModelViewerGLTFInstance extends GLTFInstance {
       const duplicateMaterial = Array.isArray(mesh.material) ?
           duplicateMaterials :
           duplicateMaterials[0];
-      const meshBack = mesh.clone() as Mesh;
-      meshBack.material = duplicateMaterial;
+      const meshBack = new Mesh(mesh.geometry, duplicateMaterial);
```

Comment:
This is reverting a change in master. Maybe a bad pull? It wasn't in your first commit.

Reply:
I see, this popped out of the merge, putting it back.